### ISSUE TITLE
Interpolation doesn't work (AreaSeries/AreaOnlySeries) in the hybrid mode.

### ISSUE THREAD

**Issue by AlimovSV** [rrag/react-stockcharts#474]
Hybrid:
![hybrid](https://user-images.githubusercontent.com/4921904/36592969-4db06872-18b1-11e8-9b49-ba436cdc38ba.png)

SVG:
![svg](https://user-images.githubusercontent.com/4921904/36592970-4dd16f22-18b1-11e8-946f-4afcf0353219.png)

```
<AreaSeries
  yAccessor={askAccessor}
  interpolation={curveStepAfter}
  stroke="#0F0"
  fill="#0F02"
/>
<AreaSeries
  yAccessor={bidAccessor}
  interpolation={curveStepAfter}
  stroke="#F00"
  fill="#F002"
/>
```

**Comment by rrag** [comment]
could you post back how it looks if you change `AreaSeries` to `LineSeries` please

**Comment by rrag** [comment]
I am guessing svg is the correct chart you want to see. is that right?

Also I would guess bid has to be `stepBefore` while ask is `stepAfter` do you agree?

**Comment by AlimovSV** [comment]
> I am guessing svg is the correct chart you want to see. is that right?

Yes SVG is a view I want to see in the hybrid mode.

> Also I would guess bid has to be stepBefore while ask is stepAfter do you agree?

You right.

> could you post back how it looks if you change AreaSeries to LineSeries please

LineSeries SVG:

![svg](https://user-images.githubusercontent.com/4921904/36626731-4bd9730e-1951-11e8-9748-ef475f35c16f.png)

LineSeries Hybrid:

![hybrid](https://user-images.githubusercontent.com/4921904/36626730-4bb70b02-1951-11e8-94cf-87b7eca27627.png)

**Comment by rrag** [comment]
something is not right, in this line series Hybrid looks correct, also svg is showing the area between the start and end points.

I looked at the code and I am not sure what is wrong. are you available now for a chat on [gitter](https://gitter.im/rrag/react-stockcharts)?

**Comment by rrag** [comment]
https://codesandbox.io/s/q9jqp20lyq

the only problem I see is that interpolation is not implemented for canvas at all. it works for svg.

However I dont see the other svg vs canvas problem you face for interpolation.

**Comment by AlimovSV** [comment]
> the only problem I see is that interpolation is not implemented for canvas at all. it works for svg.

It is exactly the issue I reported for :)

**Comment by rrag** [comment]
:) I understood it as that `stepAfter` works differently between svg and canvas.

**Comment by rrag** [comment]
fixed and published 0.7.0 (not beta, the final 0.7.0) install latest as `npm install react-stockcharts`

**Comment by AlimovSV** [comment]
@rrag  Wow! Thank you very much!
